elide-datastore/elide-datastore-hibernate5/src/main/java/com/yahoo/elide/datastores/hibernate5/HibernateStore.java

```diff
@@ -236,6 +244,92 @@ public void commit() {
         }
 
         @Override
+        public Collection filterCollection(Collection collection, String type, FilterScope<?> filterScope) {
```

Comment:
I agree. Hibernate store between 3 and 5 are actually identical. I believe the only difference is in the name of the import. We should discuss a way to make them share this code.